test/dynamo/test_activation_checkpointing.py

```diff
@@ -281,6 +297,40 @@ def fn(x, y):
         self.assertEqual(cnt.op_count, 2)
         self.assertEqual(len(cnt.graphs), 2)
 
+    @requires_cuda()
+    def test_kwargs(self):
+        def gn(x, y, z=None):
```

Comment:
What prompted this PR? Did you notice a lot of modules needing to handle kwargs?

Reply:
Yes, activation checkpointing for transfomers typically have forward methods with lots of kwargs.